apps/bfd-pipeline/bfd-pipeline-rda-grpc/src/main/java/gov/cms/bfd/pipeline/rda/grpc/sink/direct/McsClaimRdaSink.java

```diff
@@ -38,4 +41,17 @@ public long getSequenceNumberForObject(McsClaimChange object) {
     change.getClaim().setApiSource(apiVersion);
     return change;
   }
+
+  @Override
+  RdaApiClaimMessageMetaData createMetaData(RdaChange<PreAdjMcsClaim> change) {
+    final PreAdjMcsClaim claim = change.getClaim();
+    return RdaApiClaimMessageMetaData.builder()
+        .sequenceNumber(change.getSequenceNumber())
+        .claimType(RdaApiProgress.ClaimType.MCS)
+        .claimId(claim.getIdrClmHdIcn())
+        .mbiRecord(claim.getMbiRecord())
+        .claimState(nullToEmpty(claim.getIdrStatusCode()))
```

Comment:
I see how you're handling the NULL values now, but I'm worried they might send us the [empty string here](https://github.cms.gov/medicare-payment-modernization/rda-api/blob/main/proto/src/main/proto/rda/v1/mcs/mcs_claim.proto#L41) some day, which we'd want to keep seperate from a NULL

Reply:
Good catch.  The column is nullable now and this code have been changed to pass the string through without checking it for null.